wings/1.0/configuration.md

```diff
@@ -4,23 +4,24 @@
 
 ::: warning
 These are advanced configurations for Wings. You risk breaking Wings and making containers unusable if
-you configure something incorrectly. Proceed only if you know what each configuration value does.
+you misconfigure something. Proceed only if you know what each configuration value does.
 :::
 
+You must apply all changes to your wings `config.yml` file located at `/etc/pterodactyl`. Verify your config file using [Yaml Lint](http://www.yamllint.com/) should you receive errors related to YAML parsing.
```

Comment:
Wings is a proper noun in this documentation (as is Panel), so it should always be capitalized within sentences.